packages/stateless/components/NftCard.tsx

```diff
@@ -262,16 +264,29 @@ export const NftCard = forwardRef<HTMLDivElement, NftCardProps>(
               </div>
             )}
 
-            {/* Floor price */}
-            {floorPrice && (
+            {/* Highest offer */}
+            {highestOffer && (
               <div className="space-y-2">
-                <p className="secondary-text">{t('title.floorPrice')}</p>
-
+                <p className="secondary-text">{t('title.highestOffer')}</p>
                 <p className="body-text font-mono">
-                  {floorPrice.amount.toLocaleString(undefined, {
+                  {highestOffer.amount?.toLocaleString(undefined, {
                     maximumSignificantDigits: 3,
                   })}{' '}
-                  ${floorPrice.denom}
+                  ${highestOffer.denom}
```

Comment:
can you replace this with a `TokenAmountDisplay` component like the USD value below? that way it will render a token icon and look consistent. see my other comment about storing the token metadata directly in `highestOffer` so you can access its information here.